src/proxy-ui-api/frontend/src/views/Clients/Services/AddRestDialog.vue

```diff
@@ -1,13 +1,32 @@
 <template>
   <simpleDialog
     :dialog="dialog"
+    :width="560"
     title="services.addRest"
     @save="save"
     @cancel="cancel"
     :disableSave="!isValid"
   >
     <div slot="content">
+
       <ValidationObserver ref="form" v-slot="{ validate, invalid }">
+        <div class="dlg-edit-row">
+          <div class="dlg-row-title">{{$t('services.serviceType')}}</div>
+
+          <ValidationProvider
+            rules="required"
+            name="serviceType"
+            v-slot="{ errors }"
+            class="validation-provider dlg-row-input">
+
+            <v-radio-group v-model="serviceType" name="serviceType" :error-messages="errors" row>
```

Comment:
This causes [vue-i18n] Value of key 'fields.serviceType' is not a string!  warning in browser console. 
In en.json file there is "fields"  group that should have text for "serviceType".  That is where i18n gets the localisations for the field names.
I don't know in what situation the text would actually be visible in the ui, but that would fix the warning.